docs/cli/README.md

```diff
@@ -113,39 +113,62 @@ apiVersion: k8s.kolide.com/v1alpha1
 kind: OsqueryOptions
 spec:
   config:
-    distributed_interval: 3
-    distributed_tls_max_attempts: 3
-    logger_plugin: tls
-    logger_tls_endpoint: /api/v1/osquery/log
-    logger_tls_period: 10
+    options:
```

Comment:
Why are `options` nested? Anything else that could be used in config which will not be an option?

Reply:
Yes, other top-level keys are related to FIM and Yara. See the darwin override below for example.